### DISCUSSION TITLE
Opportunities for opensource documentation

### DISCUSSION THREAD

**Discussion by andrewjka** [GitbookIO/community#267] [Feature Requests]
Hey, I see that the books functionality is the same as regular github as well. So why can't ordinary users who are not book authors submit an edit request ? - That would be really cool.

**Comment by aikoii** [comment]
Hey @HallisCode! While we don't allow for external contributions in GitBook just yet, if your spaces are synced with GitHub and are public you can actually display a link to edit on GitHub. You can do so in Space Customization > Integrations. It takes you to the GitHub file, and there users can contribute to your docs :slightly_smiling_face:

**Reply by natanfudge** [reply]
I think being able to edit in gitbook would be awesome, because the github editor is completely barebones.
